plugins/simplechart-dev-mode/inc/settings-page.php

```diff
@@ -2,7 +2,14 @@
 /**
  * FM submenu page with setting for overriding plugin js
  */
+
+/**
+ * Class for dev mode settings
+ */
 class Simplechart_Dev_Mode_Settings {
+	/**
+	 * @var instance
```

Comment:
Add Simplechart_Dev_Mode_Settings as a type to hint that instance is an instance of this class.